### PR TITLE
Feature/user menu flyout

### PR DESCRIPTION
**Pull request by frankdsm**
## PR Checklist

This PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] A changelog entry has been added to CHANGELOG.md if necessary

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] react application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@acpaas-ui/react-components",
-  "version": "1.1.2",
+  "version": "1.3.3",
```
Comment (TriangleJuice) [inline package.json]: LOL

File: packages/user-menu/src/UserMenu.js [hunk 2]
```diff
@@ -1,48 +1,108 @@
...
-        Uitloggen
-        </Button>
-      </FlyoutButton>
-  )
+
+class UserMenu extends Component<Props> {
+  renderAvatar() {
+    return (
+      <Avatar
+        image={ this.props.avatarUrl || 'https://www.antwerpen.be/assets/overzicht/gfx/no-thumbnail.svg' }
```
Comment (TriangleJuice) [inline packages/user-menu/src/UserMenu.js]: I don't think pointing to an image is a good idea. Instead, you should use the `fa-user` icon, like demoed here: https://a-ui.github.io/core_branding_scss/#atoms-avatar

File: packages/user-menu/src/UserMenu.spec.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      firstName="John"
+      lastName="Doe"
+      loggedIn={true}
+      logoutUrl="/auth/logout/aprofiel">
+      <div>This is the content</div>
+    </UserMenu>);
+    expect(component.find('.m-flyout')).toBeDefined();
+  });
+
+  // test('Flyout is rendered when trigger is passed', () => {
```
Comment (TriangleJuice) [inline packages/user-menu/src/UserMenu.spec.js]: What are you going to do with these comments?

File: styleguide.config.js [hunk 4]
```diff
@@ -33,7 +33,7 @@ module.exports = {
         {
           rel: 'stylesheet',
           href:
-            'https://cdn.antwerpen.be/core_branding_scss/3.0.1/main.min.css'
+            'https://cdn.antwerpen.be/core_branding_scss/3.0.3/main.min.css'
```
Comment (TriangleJuice) [inline styleguide.config.js]: `3.1.0` is the latest version now.

File: package.json (hunk 1 above)
Comment (frankdsm) [reply package.json]: There were some version bumps already apparently

File: packages/user-menu/src/UserMenu.spec.js (hunk 3 above)
Comment (frankdsm) [reply packages/user-menu/src/UserMenu.spec.js]: Fix them ;)

File: packages/user-menu/src/UserMenu.js [hunk 5]
```diff
@@ -59,51 +71,52 @@ class UserMenu extends Component<Props> {
...
-      <div style={{ textAlign: 'right' }}>
-        <Button iconLeft="user">Aanmelden</Button>
-      </div>);
+    return (<Button iconLeft="user">Aanmelden</Button>);
   }
 
   render() {
-    return (this.props.loggedIn ? this.renderLoggedIn() : this.renderLoggedOut());
+    return (
+      <div style={{ textAlign: this.props.direction }}>
```
Comment (TriangleJuice) [inline packages/user-menu/src/UserMenu.js]: This is the only piece code where I wonder if it needs to be part of the component (open question).

**Pull request merged**